app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -54,7 +52,7 @@ public class MainActivity : AppCompatActivity(), ViewController, Router.Callback
 
     private var route: Route? = null;
     var locationClient: LostApiClient? = null
-    @Inject set
+        @Inject set
     var tileCache: Cache? = null
     @Inject set
```

Comment:
We need to figure out consistent style for indenting property setters and getters. The example here shows a (gasp) 2-space indent for a setter with an `@Inject` annotation. I smell as mass code style reformat coming...

http://kotlinlang.org/docs/reference/properties.html